### PR TITLE
Document how to set a hashed password

### LINKED ISSUE #220: Plain text password required for env option
The env option to set a password is a problem:
- The password is in plaintext in the docker-compose file
- The password will be saved in any bash history and docker files on the server I run my notebook on

I am aware this is for convenience, but I'm sure it is best if users hash their own passwords!

### PR DESCRIPTION
**Pull request by parente**
Already supported via the command line.
Update all the READMEs.

Fixes #220

### CODE REVIEW

**Comment by parente** [comment]
No code changes. Only READMEs.

**Comment by parente** [comment]
@kcrkor This is an alternative to overwriting the config file. Mind taking a look to see if the doc changes make sense?

**Comment by parente** [comment]
@jakirkham do these doc changes look OK to you?

File: all-spark-notebook/README.md [hunk 1]
```diff
@@ -195,10 +195,16 @@ Connection to Spark Cluster on Standalone Mode requires the following set of ste
...
 
 ```
-docker run -d -p 8888:8888 jupyter/all-spark-notebook start-notebook.sh --NotebookApp.base_url=/some/path
+docker run -d -p 8888:8888 jupyter/minimal-notebook start-notebook.sh --NotebookApp.password='sha1:74ba40f8a388:c913541b7ee99d15d5ed31d4226bf7838f83a50e'
+```
+
+Or to set the base URL of the notebook server:
+
+```
+docker run -d -p 8888:8888 jupyter/minimal-notebook start-notebook.sh --NotebookApp.base_url=/some/path
```
Comment (jakirkham) [inline all-spark-notebook/README.md]: Should we be specifying `all-spark-notebook`?

Comment (parente) [inline all-spark-notebook/README.md]: Yes indeed. Doc review FTW!

**Comment by jakirkham** [comment]
Yep. LGTM. Do you want more reviews or would you be happy to see it merged now?

**Comment by parente** [comment]
It's doc only, so a quick once over like you did should be enough I think. Thanks!

**Comment by kcrkor** [comment]
@parente Yes, tried it and works great and doc looks good too! Thank you!

**Comment by parente** [comment]
Thanks folks!

**Pull request merged**